rootfs/usr/lib/os-session-select

```diff
@@ -39,9 +40,18 @@ if [[ -z $SENTINEL_CREATED ]]; then
     cat "$SENTINEL_FILE"
   )
 
+  return_session=$(grep autologin-session $CONF_FILE | cut -d = -f 2)
+  return_session=$(
+    cd "$HOME"
+    mkdir -p "$config_dir"
+    cd "$config_dir"
+    echo "$return_session" > "$RETURN_SESSION_FILE"
+    cat "$RETURN_SESSION_FILE"
+  )
+
   # clear steam game desktop shortcut clutter
   DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
-  grep --files-with-matches "Exec=steam steam://rungameid/" ${DATA_HOME}/applications/* | tr '\n' '\0' | xargs -0 -I {} rm {} || true
+  grep --files-with-matches "Exec=steam steam://rungameid/" "${DATA_HOME}"/applications/* | tr '\n' '\0' | xargs -0 -I {} rm {} || true
```

Comment:
We seem to do this twice. It's also in chimera-session. I added a bug fix to that line where I removed the * and added a -r to the beginning of the grep.